flink-formats/flink-avro/src/test/java/org/apache/flink/formats/avro/AvroRowDeSerializationSchemaTest.java

```diff
@@ -123,25 +139,66 @@ public void testDeserializeComplexRowSeveralTimes() throws IOException {
 	}
 
 	@Test
-	public void testSerializability() throws IOException, ClassNotFoundException {
-		final Tuple3<Class<? extends SpecificRecordBase>, SpecificRecord, Row> testData = AvroTestUtils.getComplexTestData();
+	public void testSpecificDeserializeFromSchemaSeveralTimes() throws IOException {
```

Comment:
Same question regarding test coverage of `UNION`, `BYTES`, `FIXED`, `INT` and `LONG`.